src/themes/material/templates/body-cell-array.tpl.html

```diff
@@ -0,0 +1,8 @@
+<ng-template key='body-cell-array.tpl.html' let-$cell let-$view="$view">
+	<ul>
+		<ng-template ngFor let-item [ngForOf]="$cell.value">
```

Comment:
why we need to use ng-template here? and why just use standard ngFor sugar syntax?